ui/src/app/edge/live/production/modal/modal.component.ts

```diff
@@ -29,34 +29,34 @@ export class ProductionModalComponent {
         this.service.getConfig().then(config => {
             this.config = config;
             let channels = [];
-            this.chargerComponents = config.getComponentsImplementingNature("io.openems.edge.ess.dccharger.api.EssDcCharger");
+            this.chargerComponents = config.getComponentsImplementingNature("io.openems.edge.ess.dccharger.api.EssDcCharger").filter(component => !component.isEnabled == false);
             for (let component of this.chargerComponents) {
                 channels.push(
                     new ChannelAddress(component.id, 'ActualPower'),
                 )
             }
             this.productionMeterComponents = config.getComponentsImplementingNature("io.openems.edge.meter.api.SymmetricMeter").filter(component => {
-                if (component.properties['type'] == "PRODUCTION") {
```

Comment:
public isProducer(component: EdgeConfig.Component) {
        if (component.properties['type'] == "PRODUCTION") {
            return true;
        } else {
            // TODO make sure 'type' is provided for all Meters
            switch (component.factoryId) {
                case 'Fenecon.Mini.PvMeter':
                case 'Fenecon.Dess.PvMeter':
                case 'Fenecon.Pro.PvMeter':
                case 'Kostal.Piko.Charger':
                case 'Kaco.BlueplanetHybrid10.PvInverter':
                case 'PV-Inverter.Solarlog':
                case 'PV-Inverter.KACO.blueplanet':
                case 'PV-Inverter.SunSpec':
                case 'SolarEdge.PV-Inverter':
                    return true;
            }
        }
        return false;
    }